src/styles/colour/index.md.njk

```diff
@@ -14,16 +14,21 @@ You must make sure that the contrast ratio of text and interactive elements in
 your service meets [level AA of the Web Content Accessibility Guidelines
 (WCAG 2.0)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
 
-## Sass variables
-Use the Sass variables listed below, so you’ll automatically get any updates to
-the GOV.UK colour palette when you update GOV.UK Frontend.
-
 ## Main colours
 
+Use the Sass variables provided rather than copying the hexadecimal colour
```

Comment:
I'd put hex in brackets because I think that's how they're more commonly referred too – hexadecimal confused me!
```suggestion
Use the Sass variables provided rather than copying the hexadecimal (hex) colour
```

Reply:
I'm also a little worried people won't know what Sass variables are. I know it's in the current copy but seeing it now without the heading I'm not sure it's clear. Perhaps we could add an example here after "...hexadecimal colour codes." that says "For example, when adding focus states, use `$govuk-focus-colour` rather than `govuk-colour("yellow")`." (Or a version of that that's technically correct.)